README.md

```diff
@@ -713,7 +713,9 @@ A curated list of awesome Java frameworks, libraries and software.
 - [Collections](http://commons.apache.org/proper/commons-collections) - Extends or augments the Java Collections Framework.
 - [DBCP](http://commons.apache.org/proper/commons-dbcp) - Database connection pooling services.
 - [Math](http://commons.apache.org/proper/commons-math) - Lightweight, self-contained mathematics and statistics components.
-- [Exec](http://commons.apache.org/proper/commons-exec) - API for dealing with external process execution and environment management in Java.
+- [Exec by Apache Commons](http://commons.apache.org/proper/commons-exec) - API for dealing with external process execution and environment management in Java.
+- [Exec by Vorburger.ch](https://github.com/vorburger/ch.vorburger.exec) - launch external processes, with significant improvements over Apache Commons Exec.
```

Comment:
This is the Apache Commons subcategory your library and zt-exec should not be included here.

Reply:
oh! Sorry, I hadn't even noticed. Updated now, proposing this under Utility, better?